static/src/js/containers/MapContainer/MapContainer.tsx

```diff
@@ -403,6 +404,22 @@ export const MapContainer: React.FC<MapContainerProps> = (props) => {
     return imageryLayersObject
   }, [colormapsMetadata, layersForProjection])
 
+  // Create an object for GIBS data keyed by collectionId
+  // if no layersForProjection, then return no GIBS data
+  const gibsDataByCollection = useMemo(() => {
+    const result: GibsDataByCollection = {}
+
+    // Check if focusedCollectionId exists and layersForProjection has elements
+    if (focusedCollectionId && layersForProjection.length > 0) {
+      result[focusedCollectionId] = {
+        layers: layersForProjection,
+        projection
+      }
+    }
```

Comment:
After seeing where projection was being used, we don't need to pass it like this. So I think this would make more sense as `gibsLayersByCollection`, instead of "data"

```suggestion
  const gibsLayersByCollection = useMemo(() => {
    const result: GibsLayersByCollection = {}

    // Check if focusedCollectionId exists and layersForProjection has elements
    if (focusedCollectionId && layersForProjection.length > 0) {
      result[focusedCollectionId] = layersForProjection
    }
```